extra/1-currency-conversion.js

```diff
@@ -15,7 +18,13 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(convertGBP) {
+  const rate = 5.7;
+  const fee = 1;
+  let convertToBRL = convertGBP * rate * ((100 - fee) / 100);
+    return convertToBRL.toFixed(2);
+}
+
```

Comment:
similar ideas apply here in the previose example:
```js
function convertToBRL(priceGBP) {
  const rate = 5.7;
  const fee = 1;
  let priceBRL = priceGBP * rate * ((100 - fee) / 100);
  return priceBRL.toFixed(2);
}
```
or alternatively, this implementation can be also readable:
```js
function convertToBRL(priceGBP) {
  return priceGBP * 5.7 * 0.99;
}
```
if you think that numbers can be confusing for other developers, you can comment your code:
```js
function convertToBRL(priceGBP) {
  // conversion rate: 5.7
  // transfer fee: 1%
  return priceGBP * 5.7 * 0.99;
}
```
that makes everything clear for those who reads your code.